source/agora/common/TransactionPool.d

```diff
@@ -105,16 +103,7 @@ public class TransactionPool
         }
     }
 
-    /***************************************************************************
-
-        Shut down the database
-
-        Note: this method must be called explicitly, and not inside of
-        a destructor.
-
-    ***************************************************************************/
-
-    public void shutdown ()
+    ~this ()
```

Comment:
Missing documentation.

Reply:
How do you imagine a good documentation for this destructor?

Reply:
Maybe just put it in the class header. "Must call GC.destroy on an instance of this class manually"

Reply:
Not agreed. Then someone will forget to remove it after this behaviour will be fixed.

Reply:
Then file an issue on github so we can track this and make a comment there that the comment which you will add should be removed.